problems/[230727] 143. Reorder List.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * leetcode problem link: https://leetcode.com/problems/reorder-list
+ */
+
+/**
+ * Definition for singly-linked list.
+ * function ListNode(val, next) {
+ *     this.val = (val===undefined ? 0 : val)
+ *     this.next = (next===undefined ? null : next)
+ * }
+ */
+/**
+ * @param {ListNode} head
+ * @return {void} Do not return anything, modify head in-place instead.
+ */
+const reorderList = function (head) {
+  let secondeNode = splitList(head);
+  secondeNode = reverseList(secondeNode);
+  mergeTwoList(head, secondeNode);
+
+  function splitList(head) {
+    let slow = head;
+    let fast = head;
+
+    while (fast && fast.next && fast.next.next) {
```

Comment:
`fast.next.next` 조건은 없어도 정상동작 할 것 같아요.
```suggestion
    while (fast && fast.next) {
```